apps/src/gamelab/GameLab.js

```diff
@@ -88,8 +88,17 @@ var GameLab = function () {
   consoleApi.setLogMethod(this.log.bind(this));
   errorHandler.setLogMethod(this.log.bind(this));
 
-  /** Expose for testing **/
-  window.__mostRecentGameLabInstance = this;
+  if (process.env.NODE_ENV !== "production") {
```

Comment:
You are right, this was an unnecessary safeguard - we may want levelbuilders creating animations on prod, and this would break that.

I'm removing this check, and have verified that the link in the 'admin' box would be available on prod too.